packages/mobile/test/values.ts

```diff
@@ -19,9 +19,12 @@ import {
   RecipientWithQrCode,
 } from 'src/recipients/recipient'
 
+export const nullAddress = '0x0'
```

Comment:
Null address is usually 20 0 bytes. Although you could compare addresses as integers (i.e. discount any leading zeros) most applications don't tend to do that.